<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f8ff;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border-radius: 8px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #45a049;
}

QLineEdit {
    border: 2px solid #ccc;
    border-radius: 4px;
    padding: 4px;
    background-color: #ffffff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Icons/logo_vit_7_3.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>C R M - Mentor Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="mentor_menu_buton_all_applications">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>	All Applications</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/preference_admin.png</normaloff>Icons/preference_admin.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mentor_menu_buton_search">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/search.png</normaloff>Icons/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="mentor_menu_lineedit_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Enter the text to search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mentor_menu_buton_preferences">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Preferences</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/prefenens.png</normaloff>Icons/prefenens.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mentor_menu_buton_exit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/exit-icon.png</normaloff>Icons/exit-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="mentor_menu_combobox">
         <item>
          <property name="text">
           <string>Tümünü Göster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Başka bir sektöre yönlendirilmeli</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bir sonraki VIT projesine katilmasi daha uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Direkt bireysel koçluk ile işe yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi ilk IT eğitimi alıp ITPH a yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi ingilizce eğitimi alıp ITPH a yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi kapsamında direkt ITPH a yönlendirilmesi uygun olur.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesinin tamamına katılması uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diger</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="mentor_menu_table_result">
      <column>
       <property name="text">
        <string>Datum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mentor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IT-Informatie</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>intensiteit</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
